### PR TITLE
zaki/welcome_page

### PR DESCRIPTION
**Pull request by zakibinary**
Changelog
---
- Show welcome screen after virtual account is created.

### CODE REVIEW

File: src/javascript/app/pages/user/new_account/welcome_page.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const BinarySocket       = require('../../../base/socket');
+const Client             = require('../../../base/client');
+const localize           = require('../../../../_common/localize').localize;
+const State              = require('../../../../_common/storage').State;
+const createElement      = require('../../../../_common/utility').createElement;
+const Url                = require('../../../../_common/url');
```
Comment (negarn) [inline src/javascript/app/pages/user/new_account/welcome_page.js]: alignment by `=`, too much empty space :)

File: src/javascript/app/pages/user/new_account/welcome_page.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            };
+
+            const welcome_msg = document.getElementById('welcome_container');
+            if (welcome_msg) {
+                welcome_msg.setVisibility(1);
+            }
+
+            if (show_welcome_msg) {
+                setButtonLink(upgrade_info.upgrade_link, 'Upgrade now');
+                // revert to upgrade_info.upgrade_link after ICO
```
Comment (negarn) [inline src/javascript/app/pages/user/new_account/welcome_page.js]: ?

Comment (zakibinary) [reply src/javascript/app/pages/user/new_account/welcome_page.js]: whoops, removed comment

File: src/javascript/app/pages/user/new_account/welcome_page.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                    const el = document.getElementById(allowed_currencies[i]);
+                    if (el) {
+                        el.classList.remove('invisible');
+                    }
+                }
+            }
+
+            const setButtonLink = (url, msg) => {
+                if (upgrade_btn) {
+                    upgrade_btn.html(createElement('span', { text: localize(msg) })).setAttribute('href', Url.urlFor(url));
```
[the remark this answers is not in the document]
Comment (kellybinary) [reply src/javascript/app/pages/user/new_account/welcome_page.js]: ahh i see, ok then :)

**Pull request merged**